libs/spark/stories/navbar.stories.tsx

```diff
@@ -76,9 +72,7 @@ const Template: Story<NavBarProps> = (args) => (
       >
         0
       </InboxNavBarButton>
-      <UserMenu>
-        <UserDuotone color="onLight" fontSize="large" />
-      </UserMenu>
+      <Avatar src="/img/student-boy-2.png" alt="john doe" size="medium" />
```

Comment:
Where is this src referencing? Does it need to be added?

Reply:
Assets under the `/libs/spark/public` folder. Webpack loads those files at static assets by default. You can see them working in the storybook.